Products/Zuul/routers/manufacturers.py

```diff
@@ -53,11 +56,15 @@ def editProduct(self, params=None):
         """
         facade = self._getFacade()
         oldData = facade.getProductData(params['uid'], params['oldname'])[0]
-        facade.editProduct(params)
-
+        scheduled = facade.editProduct(params)
         audit('UI.Manufacturers.editProduct', params['uid'],
               data_=params, oldData_=oldData)
-
+        if scheduled:
+            return DirectResponse.succeed(
+                "Job to edit product and update product class for its "
+                "instances has been scheduled. Wait for finish to add "
+                "more changes to this product."
```

Comment:
A shorter version of the text can read:

> Scheduled job to update product %s.

where `%s` is the name of the product.  Use the name from the old data as that's what is being updated.